.env.example

```diff
@@ -14,3 +14,4 @@ SALT_ROUNDS=2
 AZURE_STORAGE_CONNECTION_STRING=<AZURE_STORAGE_CONNECTION_STRING>
 AZURE_STORAGE_ACCOUNT_NAME=<AZURE_STORAGE_ACCOUNT_NAME>
 PRODUCTION_WEB_URL=
+SERVER_URL=
```

Comment:
Perhaps we can add a comment here with the default  `SERVER_URL` for future reference!